bidwire/bid.py

```diff
@@ -17,36 +17,36 @@ class Bid(Base):
     location = Column(Text)
     open_date = Column(DateTime)
     items = Column(JSON)
+    site = Column(Text)
```

Comment:
Thanks for the suggestions! I wasn't able to get the Bid class imported in the alembic file, so left it as a string for now.